src/store/flow/block-types/MobilePrimitives_NumericResponseBlockStore.ts

```diff
@@ -13,50 +13,52 @@ export const getters: GetterTree<IFlowsState, IRootState> = {}
 export const mutations: MutationTree<IFlowsState> = {}
 
 export const actions: ActionTree<IFlowsState, IRootState> = {
-  async setValidationMinimum({commit, rootGetters}, {blockId, value} : {blockId: string, value: number}) {
+  async setValidationMinimum({ commit, rootGetters }, { blockId, value }: {blockId: string; value: number}) {
     commit('flow/block_updateConfigByKey', {
       blockId,
       key: 'validationMinimum',
-      value
-    }, {root: true})
+      value,
+    }, { root: true })
     return value
   },
-  async setValidationMaximum({commit, rootGetters}, {blockId, value} : {blockId: string, value: number}) {
+  async setValidationMaximum({ commit, rootGetters }, { blockId, value }: {blockId: string; value: number}) {
     commit('flow/block_updateConfigByKey', {
       blockId,
       key: 'validationMaximum',
-      value
-    }, {root: true})
+      value,
+    }, { root: true })
     return value
   },
-  async setMaxDigits({commit, rootGetters}, {blockId, value} : {blockId: string, value: number}) {
+  async setMaxDigits({ commit, rootGetters }, { blockId, value }: {blockId: string; value: number}) {
```

Comment:
I think we also use a comma style here in flow-runner. Is that right @bzabos ?

Reply:
Again, not sure where this rule is getting applied from. It might be from one of these 
    'plugin:vue/essential',
    '@vue/airbnb',
    '@vue/typescript/recommended'

Reply:
I'm happy to adopt this standard going forward unless there are any objections